include/fmt/core.h

```diff
@@ -87,6 +87,9 @@
 #define FMT_HAS_CPP17_ATTRIBUTE(attribute) \
   (FMT_CPLUSPLUS >= 201703L && FMT_HAS_CPP_ATTRIBUTE(attribute))
 
+#define FMT_HAS_CPP20_ATTRIBUTE(attribute) \
+  (FMT_CPLUSPLUS >= 202002L && FMT_HAS_CPP_ATTRIBUTE(attribute))
```

Comment:
I don't think we need this for a single check esp. since MSVC requires special handling. Let's just inline in the call site.

Reply:
Would it not be better to create this feature now, in the event that the other C++20 attributes (likely, unlikely) are desired in the future?

Reply:
There is little value in these variations of `FMT_HAS_CPP_ATTRIBUTE` so let's just inline.